frontend/src/features/properties/map/acquisition/AcquisitionContainer.tsx

```diff
@@ -53,7 +67,22 @@ export const AcquisitionContainer: React.FunctionComponent<IAcquisitionContainer
       header={<AcquisitionHeader acquisitionFile={acquisitionFile} />}
       footer={null}
     >
-      <></>
+      <>
```

Comment:
There isn't really a slot for this button and creating a layout for acquisition files is outside the scope of this story. I may need to circle back later in the sprint and make sure this ends up in the right place.